package/spec/frontend/media_player/volume_binding_spec.js

```diff
@@ -91,8 +90,7 @@ describe('volumeBinding', function() {
     it('does not fade in until promise returned by play is resolved', function() {
       var player = fakePlayer();
       var settings = new Backbone.Model({volume: 98});
-      var deferred = new jQuery.Deferred();
-      player.play = function() { return deferred.promise(); };
+      player.play = function() { return Promise.resolve() };
```

Comment:
Without looking at any details here, I just want to point out that this changes the test in an important war: Before the promise stayed pending. We should again be sure that this fails for the right reason.